ogre_mesh_viewer.py

```diff
@@ -313,6 +313,63 @@ def preRenderTargetUpdate(self, evt):
             entity.getSubEntities()[highlight].setMaterial(self.app.highlight_mat)
             self.highlighted = highlight
 
+        if ImGui.CollapsingHeader("Transform"):
+            flags = ImGui.TableFlags_Borders | ImGui.TableFlags_SizingStretchProp
+
+            enode = entity.getParentSceneNode()
+
+            ImGui.Text("Position")
+            if ImGui.BeginTable("Position", 3, flags):
```

Comment:
I think using Tables here is too bloated both in terms of code and in terms of UI space.

Can you change this to BulletText as in "Bounds"?

Reply:
Most game engines show this in tabbed format and even with different colors for each axis.

I think the Bulleted format looks bad

Reply:
please go with the bulleted format for now. I have an idea how to improve formatting closer to

unity:
![grafik](https://github.com/OGRECave/ogre-meshviewer/assets/4941802/843e2193-b218-41a2-b510-b5c7c986ebe8)

and unreal:
![grafik](https://github.com/OGRECave/ogre-meshviewer/assets/4941802/111065c6-a477-4132-bbf2-47f1c4ddffe8)

I will apply that to transform and bounds in a second step